docs/tutorials/initializedirtyimage.py

```diff
@@ -0,0 +1,190 @@
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# + nbsphinx="hidden"
+# %matplotlib inline
+
+# + nbsphinx="hidden"
+# %run notebook_setup
+# -
+
+# # Not completely done, still have to do some links do some fleshing out, just want to get it here to make sure there isn't anything critically wrong.
+#
+# ## Initializing Model with the Dirty Image
+#
+# In this tutorial we will be going over how to optimize the model based off of the dirty image. It has a low loss value (maximal likelihood) and so this allows the optimization to be much quicker. The problem with the dirty image is that certain model paramterizations enforce image positivity while the dirty image usually contains regions of negative flux. So in this tutorial we will be optimizing the model to be similar to the dirty image (phrasing could be better here). We will also be going over saving and loading the model. After saving and loading it, it can be then optimized against the visibility data to complete it (though this will not be done in this tutorial). We will be using the dataset of the ALMA logo first used in the Gridding and Diagnostic Images tutorial.
+#
+#
+
+# ### Image Setup
+#
+# Here we will be setting up the ALMA Logo image dataset and displaying it. Consult the Gridding and Diagnostic Images Tutorial for reference.
+
+import torch
+import numpy as np
+import matplotlib.pyplot as plt
+from mpol import gridding, coordinates, precomposed, losses, utils
+from astropy.utils.data import download_file
+from torch.utils.tensorboard import SummaryWriter
+
+# In this tutorial we will be saving and loading the model we use and it is important to note that when saving and loading a model the cell_size, nchan, and npix are the same between the model and the coordinates if used in another document.
+
+# +
+# load the mock dataset of the ALMA logo
+fname = download_file(
+    "https://zenodo.org/record/4498439/files/logo_cube.npz",
+    cache=True,
+    show_progress=True,
+    pkgname="mpol",
+)
+
+# this is a multi-channel dataset... but for demonstration purposes we'll use
+# only the central, single channel
+chan = 4
+d = np.load(fname)
+uu = d["uu"][chan]
+vv = d["vv"][chan]
+weight = d["weight"][chan]
+data_re = d["data_re"][chan]
+data_im = -d["data_im"][
+    chan
+]  # we're converting from CASA convention to regular TMS convention by complex conjugating the visibilities
+
+# define the image dimensions, making sure they are big enough to fit all
+# of the expected emission
+coords = coordinates.GridCoords(
+    cell_size=0.03, npix=180
+)  # The smaller the cell size and the larger the npix can greatly increase run time
+gridder = gridding.Gridder(
+    coords=coords,
+    uu=uu,
+    vv=vv,
+    weight=weight,
+    data_re=data_re,
+    data_im=data_im,
+)
+
+# export to PyTorch dataset
+dset = gridder.to_pytorch_dataset()
+# -
+
+# Show the dirty image
+img, beam = gridder.get_dirty_image(weighting="briggs", robust=0.0, unit="Jy/beam")
```

Comment:
It would be useful to accentuate the negative values in the dirty image using a diverging colormap: https://matplotlib.org/stable/tutorials/colors/colormaps.html